<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Idling</class>
 <widget class="QWidget" name="Idling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="bots_layout">
   <property name="topMargin">
    <number>35</number>
   </property>
   <property name="bottomMargin">
    <number>35</number>
   </property>
   <item>
    <widget class="QPushButton" name="idling_list">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Lists priority idling queue of current bot instance.</string>
     </property>
     <property name="text">
      <string>List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="idling_add">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Adds given appIDs to priority idling queue of current bot instance.</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="idling_remove">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Removes given appIDs from priority idling queue of current bot instance.</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
